sql/2024/privacy/ads_and_sellers_graph.sql

```diff
@@ -0,0 +1,114 @@
+WITH RECURSIVE pages AS (
+  SELECT
+    CASE page -- publisher websites may redirect to an SSP domain, and need to use redirected domain instead of page domain
+      WHEN 'https://www.chunkbase.com/' THEN 'cafemedia.com'
```

Comment:
Why only one example of this? Are they they only publisher redirecting?

Reply:
Such verification required going and checking it manually.
That's why I added a redirect attribute in https://github.com/HTTPArchive/custom-metrics/pull/136.

So it was rather a note for the further analysis.

Reply:
Maybe a note to say "One example is included here, but hope to expand that in future years with results of https://github.com/HTTPArchive/custom-metrics/pull/136"?

Just looks a bit weird here there only being one and looks like we might have forgotten to fill in the rest.